test/sql/aggregate/aggregates/test_max_intersections.test

```diff
@@ -0,0 +1,107 @@
+# name: test/sql/aggregate/aggregates/test_max_intersections.test
```

Comment:
Can we add tests for windowing (OVER clause)? I suspect you will can't a custom window function.